### ISSUE TITLE
Install.exe, hdsetup.exe, and hdsetup.com are v4 source code.

### ISSUE THREAD

**Issue by the-grue** [roelandjansen/pcmos386v501#65]
Built everything and wrote it to a floppy image and started to put it through testing and discovered the files listed above are from v4 MOS, not v5.  These do not create v5 DOS compatible disks that msys can lay a boot sector down on, they try to create the older MOS large disk volumes.

Anyway, another issue, another project for someone who would rather dabble in C than Assembler.

**Comment by roelandjansen** [comment]
I'll double check the tapes when time permits, again to be sure that nothing is wrong there.

**Comment by the-grue** [comment]
Thanks @roelandjansen!  I can proceed with my work, so no rush.

**Comment by the-grue** [comment]
Disregard.  I figured out what the problem was.  With hdsetup.com being built as one of the utilities it is chosen over hdsetup.exe.  hdsetup.com is v4 and hdsetup.exe is v5.  I will upload a fix.

**Comment by the-grue** [comment]
See #70

**Comment by andrewbird** [comment]
So if I understand correctly, there is a prebuilt copy of hdsetup v5, and the sources we have are v4?

**Comment by andrewbird** [comment]
Ahh, it seems there's an hdsetup.c in the cutils directory to be built with Borland C. Reading the source it requires to be run on v5 or later, so I suspect this might be the version of source you were looking for.

**Comment by the-grue** [comment]
> Ahh, it seems there's an hdsetup.c in the cutils directory to be built with Borland C. Reading the source it requires to be run on v5 or later, so I suspect this might be the version of source you were looking for

Yeah, I had built hdsetup.exe, install.exe, and export.exe out of cutils.  There is a hdsetup.asm that builds hdsetup.com in the various kernel directories.  It is built by default as part of "make -f makeutil.mak all"

MOS seems to have a peculiarity where if there is a program with the same name but .com vs .exe extension in the directory you are working in or path to it will choose the .com version even if you specifically call the .exe version.

This is what was throwing me off.  I would run install and it would run a v4 version of hdsetup.  (hdsetup.com)  If I call hdsetup it would run hdsetup.com.  If I call hdsetup.exe, it would run hdsetup.com.

So I patched to remove the build of hdsetup.com out of makeutil.mak in the latest directory and it works fine now.
